src/openrct2/ride/RideTypes.h

```diff
@@ -17,8 +17,7 @@
 using ride_id_t = uint16_t;
 struct Ride;
 
-constexpr const ride_id_t RIDE_ID_NULL = 0xFF; // std::numeric_limits<ride_id_t>::max();
-constexpr const ride_id_t RideIdNewNull = std::numeric_limits<ride_id_t>::max();
+constexpr const ride_id_t RIDE_ID_NULL = std::numeric_limits<ride_id_t>::max();
```

Comment:
Would it be possible to change this to `RideIdNull` or something not in all caps.

Reply:
We seem to use a mishmash of TitleCase and SCREAMING_SNAKE_CASE for constants. Would be good to have an agreed-upon standard. (My preference happens to go towards the latter to make clear it's a constant.)